.github/workflows/fetch.yml

```diff
@@ -0,0 +1,43 @@
+name: Integration
+
+on:
+  push:
```

Comment:
I think that pull_request is more suitable than push in terms of verifying changes before the master branch changes.